src/styles/partials/components/_modal.scss

```diff
@@ -0,0 +1,151 @@
+.dg_modal {
```

Comment:
General functionality question. When opened, should the background still scroll? If not, recommend appending a string to the js file to write an `open` class, or something along those lines to the body. Ie. `document.body.classList.add("open");`. The corresponding class could look something like this: `body.open {overflow: hidden;}`. This would prevent the background content from scrolling.

Reply:
@theilman7 the background would not scroll, but it is possible the modal would scroll with really long content. See [here](https://getbootstrap.com/docs/4.3/components/modal/).

![image](https://user-images.githubusercontent.com/1933400/67202575-ef9b6400-f3d6-11e9-9c9a-9921a1a4e64f.png)

Reply:
@martypowell , i was not able to scroll with the solution you provided but I added overflow : auto in my .dg_modal__inner class which shows the scrollbar when text is larger on a dialog box. pleae let me know if this solution is accetable.